Assets/00_Content/Scripts/Interactables/PickUp.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Interactables {
+
+	public class PickUp : MonoBehaviour, IPickUp {
```

Comment:
Same question here as with Interactable and IInteractable. Isn't an interface a bit redundant?